test/membrane_opus/encoder_test.exs

```diff
@@ -0,0 +1,91 @@
+defmodule Membrane.Opus.Encoder.EncoderTest do
+  use ExUnit.Case, async: true
+
+  alias Membrane.Opus.Encoder
+  alias Membrane.Opus.Support.Reader
+  alias Membrane.Caps.Audio.Raw
+  import Membrane.Testing.Assertions
+  alias Membrane.Testing
+  alias Membrane.Element.File
+  import Membrane.ParentSpec
+
+  @in_path "input.wav"
+  @sample_raw_packets Reader.read_packets("test/fixtures/raw_packets")
+  @sample_opus Reader.read_packets("test/fixtures/opus_packets")
+
+  setup do
+    elements = [
+      source: %File.Source{
+        location: @in_path
+      },
+      opus: %Encoder{
+        application: :audio,
+        input_caps: %Raw{
+          channels: 2,
+          format: :s16le,
+          sample_rate: 48_000
+        }
+      },
+      sink: %File.Sink{location: "output.opus"}
+    ]
+
+    links = [link(:source) |> to(:opus) |> to(:sink)]
+
+    {:ok, pipeline_pid} =
+      Testing.Pipeline.start_link(%Testing.Pipeline.Options{
+        elements: elements,
+        links: links
+      })
+
+    {:ok, pipeline_pid: pipeline_pid}
+  end
+
+  test "writing files", state do
+    assert :ok = Membrane.Pipeline.play(state[:pipeline_pid])
+    assert_start_of_stream(state[:pipeline_pid], :sink)
+
+    assert_end_of_stream(state[:pipeline_pid], :sink)
```

Comment:
this is consistently failing. i'm not sure why. is there some reason why the pipeline would just hang?